libcxx/test/std/language.support/support.dynamic/new.delete/new.delete.array/sized_delete_array14.pass.cpp

```diff
@@ -15,6 +15,9 @@
 // XFAIL: apple-clang
 // XFAIL: using-built-library-before-llvm-11
 
+// AIX, z/OS, and MinGW default to -fno-sized-deallocation.
+// XFAIL: target={{.+}}-aix{{.*}}, target={{.+}}-zos{{.*}}, target={{.+}}-mingw32{{.*}}
```

Comment:
The patterns for mingw here should be `{{.*}}-windows-gnu` - the triples get normalized into that form. Please grep for that and see which exact spelling we use elsewhere (can’t easily check myself ATM).

Reply:
(Or `{{.+}}` for the arch obviously is fine too.)

Reply:
You are right. Targets used for MinGW are as follows. I will change to use `{{.+}}-windows-gnu` as suggested. Thanks, @mstorsjo!
```
target=x86_64-w64-windows-gnu
target=i686-w64-windows-gnu
```